R/hms.R

```diff
@@ -274,3 +274,39 @@ print.hms <- function(x, ...) {
   cat(format(x), sep = "\n")
   invisible(x)
 }
+
+# Sequence generation -----------------
+
+#' @rdname hms
+#' @inheritParams base::seq
+#' @export
+seq.hms <- function(
+    from = hms(1),
+    to = hms(1),
+    ...) {
```

Comment:
Minor nit: we prefer single indent these days.

```suggestion
seq.hms <- function(
  from = hms(1),
  to = hms(1),
  ...
) {
```

Is there a reason to not add `by` to the signature to avoid handling the dots?

Reply:
I refactored the code to have by in the signature and get rid of ... handling.